plugins/linux-v4l2/v4l2-output.c

```diff
@@ -0,0 +1,160 @@
+#include <obs-module.h>
+#include <util/platform.h>
+#include <linux/videodev2.h>
+#include <sys/ioctl.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#define MAX_DEVICES 32
+
+struct virtualcam_data {
+	obs_output_t *output;
+	int device;
+	uint32_t frame_size;
+};
+
+static const char *virtualcam_name(void *unused)
+{
+	UNUSED_PARAMETER(unused);
+	return "Virtual Camera Output";
+}
+
+static void virtualcam_destroy(void *data)
+{
+	struct virtualcam_data *vcam = (struct virtualcam_data *)data;
+	close(vcam->device);
+	bfree(data);
+}
+
+static bool loopback_module_loaded()
+{
+	bool loaded = false;
+
+	FILE *fp = popen("lsmod | grep v4l2loopback", "r");
+	int status = pclose(fp);
+
+	if (WEXITSTATUS(status) == 2)
+		loaded = true;
+
+	return loaded;
+}
+
+static void loopback_module_load()
+{
+	system("pkexec modprobe v4l2loopback exclusive_caps=1 card_label='OBS Virtual Camera'");
```

Comment:
This should be named "OBS Video Output" to be consistent with Windows.

Reply:
Actually, Windows needs to be changed in this case. "OBS Virtual Camera" is correct.